packages/mattermost-redux/src/reducers/entities/search.ts

```diff
@@ -322,6 +322,18 @@ function isLimitedResults(state = -1, action: GenericAction): number {
     }
 }
 
+function recentSearches(state: Record<string, Search[]> = {}, action: GenericAction) {
+    const {data, type} = action;
+
+    switch (type) {
+    case SearchTypes.RECENT_SEARCHES_SUCCESS: {
+        return data;
+    }
+    default:
```

Comment:
Could you add a case to reset this state on `UserTypes.LOGOUT_SUCCESS`? Most of our reducers have that, so I'm not sure why some of the others in this file are missing it